.github/workflows/notify-discord.yml

```diff
@@ -0,0 +1,24 @@
+name: Notify Discord
+
+on:
+  push:
+    tags:
+      - '@faustwp/wordpress-plugin@*'
+      - '@faustwp/core@*'
+      - '@faustwp/cli@*'
+      - '@faustwp/blocks@*'
+      # Test this action without creating a full release.
+      - '@faustwp/test@*'
```

Comment:
You can manually test this action and see it notify the newly created [github-updates](https://discord.com/channels/836253505944813629/1114240986755911730) channel

Reply:
Would you be able to explain further? I pushed a new tag, `'@faustwp/test-for-pr@*'`, but I don't see any notification in the discord channel.